region/norfair/east.json

```diff
@@ -1580,9 +1735,29 @@
           "unlock": [ "canOpenRedDoors" ],
           "sparking": {
             "runways": [
+              {
+                "length": 7,
```

Comment:
What does this smaller runway represent?

Reply:
Coming in from the Right door running up to the crumbleblocks and charging back in.

Reply:
Thought it might be that, but wasn't sure because there was no corresponding change on the other end.